### ISSUE TITLE
Missing root key returns undefined

### ISSUE THREAD

**Issue by rdominy** [mrodrig/doc-path#3]
The documentation for evaluatePath states that null is returned if the key does not exist.  That is only true for a nested key.  A root level key returns undefined.  Example:

~~~~javascript
var DocPath = require('doc-path');

var myDoc = {
	foo: true
};

console.log(DocPath.evaluatePath(myDoc,'nosuch'));
console.log(DocPath.evaluatePath(myDoc,'nosuch.thing'));

~~~~

This outputs:
undefined
null

For my usage, returning undefined in all cases would be more helpful because null might be a valid document value.  But I understand that would be a breaking change unless you did some sort of module preference (e.g., DocPath.setMissingDefault(undefined)).

**Comment by mrodrig** [comment]
Thanks for reporting this and sorry for the delay in getting back to you.  I'll comment back once it is fixed and the new version is published to NPM.

**Comment by mrodrig** [comment]
It should be fixed in `1.1.0` on NPM now.  Thanks again and sorry for the delay!
